app-text/dblatex/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<longdescription lang="en">
+	</longdescription>
+</pkgmetadata>
+
```

Comment:
I would rather wait first to see if somebody else wants to maintain it.

In principal I'm not against the idea of being a proxy-maintainer, but I'd have to look into what's involved first.  I don't want to promise something I can't keep.